teuthology/test/task/__init__.py

```diff
@@ -146,6 +146,7 @@ def test_setup_called(self):
             setup=DEFAULT,
             begin=DEFAULT,
             end=DEFAULT,
+            teardown=DEFAULT,
```

Comment:
And because I am thick, patching these to DEFAULT just means they are patched to Mocks, so they won't be doing anything, but the assert_called_* still prove they made it?   and you need to patch teardown for completeness, because it's pass anyway, and isn't asserted about here, but both those could change in future?